core/field_label.js

```diff
@@ -36,12 +36,16 @@ goog.require('goog.math.Size');
 
 /**
  * Class for a non-editable, non-serializable text field.
- * @param {string} text The initial content of the field.
+ * @param {!string} text The initial content of the field.
  * @param {string=} opt_class Optional CSS class for the field's text.
  * @extends {Blockly.Field}
  * @constructor
  */
 Blockly.FieldLabel = function(text, opt_class) {
+  if (!text) {
```

Comment:
This will be false for the empty string as well, which I don't think is a behavior change you intended to make. 

In general, why are you adding this check? It looks like setText will handled it by casting to a String if it's not null. Is it a problem with validators?

Reply:
You're right, Fixed =)